katello_38.dot

```diff
@@ -0,0 +1,346 @@
+@startuml
+
+title <b>Satellite BIG Picture</b>
+
+[Browser] as browser
+
+package "Content Host" {
+	[Yum] as yum
+	[RHSM / Subscription Manager] as rhsm
+	[Katello Agent / Goferd] as katello_agent
+	[Puppet Agent] as puppet
+	[Foreman SCAP Client] as foreman_scap_client
+	[Insights] as insights_client
+}
+
+frame "Satellite Server" {
+
+'	interface Waldirio {
+'		Listening on hostname:8080
+'	}
+
+	database "Server DB" {
+		[Postgres\n- Listen on localhost:5432] as postgres
+		[MongoDB\n- Listen on localhost:27017] as mongo_server
+	}
+	
+	
+	[Qpid\n- Listen on localhost:5671\n- Listen on localhost:5672] as qpidd_server
+	[Qdrouterd / Dispatch Router\n- Listen on hostname:5646\n- Listen on hostname:5647] as qdrouterd_server
+	[Smart Proxy / Foreman Proxy\n- Listen on hostname:9090] as server_smart_proxy
+	[Smart Proxy Dynflow Core(REX)\n- Listen on hostname:8008] as server_smart_proxy_dynflow_core
+	[TFTP\n- Listen on hostname:69] as server_tftp
+	[DNS\n- Listen on hostname:53] as server_dns
+	[DHCP] as server_dhcp
+
+	frame "Tomcat" as tomcat_server {
+	  [Candlepin\n- Listen on hostname:8080\n- Listen on hostname:8443\n- Listen on hostname:8009\n- Listen on localhost:8005] as candlepin
+	}
+
+	frame "Streamer" as streamer_server {
+		[Pulp Streamer\n- Listen on localhost:8751] as pulp_server_streamer
+		[Squid\n- Listen on hostname:3128] as server_squid
+		[WSGI App] as server_wsgi_streamer
+	}
+	
+	frame "Pulp" as pulp_server {
+		[API] as pulp_server_api
+		[Pulp Worker] as pulp_server_worker
+		[Pulp Resource Manager] as pulp_server_resource_manager
+		[Pulp Celery Beat] as pulp_server_celery_beat
+	}
+
+	frame "Apache" as server_apache {
+		[Puppet Master] as puppet_master_server
+		[Mod WSGI] as mod_wsgi
+		[Mod WSGI pulp-cont] as mod_wsgi_pulp_cont
+		[Mod WSGI pulp-forg] as mod_wsgi_pulp_forg
+
+		frame "Foreman Application" {
+			[Foreman] as foreman
+			[Katello] as katello
+			[Foreman Tasks / Dynflow] as foreman_tasks
```

Comment:
I would not put `dynflow` here as it is a separate process while `foreman_tasks` is a plugin that provides UI and other mechanisms.